src/Microsoft.OpenApi/Writers/OpenApiJsonWriter.cs

```diff
@@ -153,23 +153,23 @@ public override void WriteEndArray()
         /// </summary>
         /// <param name="name">The property name.</param>
         /// public override void WritePropertyName(string name)
-        public override void WritePropertyName(string name)
+        public override void WritePropertyName(string? name)
```

Comment:
the name is non nullable in the parent interface, align and add defensive statements